memcached-operator/README.md

```diff
@@ -0,0 +1,59 @@
+# Memcached Operator
+
+## Overview
+This Memcached operator is a simple example operator for the [Operator SDK][operator_sdk] and includes some basic end-to-end tests.
+
+## Quick Start
+This quick start guide walks through the process of building the memcached-operator and running its end-to-end tests.
+
+### Prerequisites
+- [dep][dep_tool] version v0.5.0+.
+- [go][go_tool] version v1.10+
+- [docker][docker_tool] version 17.03+
+- Access to a kubernetes v1.9.0+ cluster
+
+### Install the Operator SDK CLI
+First, checkout and install the operator-sdk CLI:
+```
+$ cd $GOPATH/src/github.com/operator-framework/operator-sdk
+$ git checkout master // currently, there are no releases that include the test framework, so use the master for now
+$ dep ensure
+$ go install github.com/operator-framework/operator-sdk/commands/operator-sdk
+```
+
+### Initial Setup
+Checkout this Memcached Operator repository
+```
+$ mkdir $GOPATH/src/github.com/operator-framework
```

Comment:
This should already be present if they installed the sdk binary from `github.com/operator-framework/operator-sdk` in the previous step. So we can remove this.